docs/content/posts/2025-10-24-release-2.0.1.md

```diff
@@ -0,0 +1,200 @@
+---
+title:  "Apache Flink 2.0.1 Release Announcement"
+date: "2025-10-24T00:00:00.000Z"
+aliases:
+- /news/2025/10/24/release-2.0.1.html
+authors:
+- zakelly:
+  name: "Zakelly Lan"
+
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink 2.0 series.
+
+This release includes 52 bug fixes, vulnerability fixes, and minor improvements for Flink 2.0.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12355557).
+
+We highly recommend all users upgrade to Flink 2.0.1 or higher.
+
+# Release Artifacts
+
+## Maven Dependencies
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>2.0.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>2.0.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>2.0.1</version>
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink/tags?page=1&name=2.0.1) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=2.0.1) (ASF repository)
+
+## PyPi
+
+* [apache-flink==2.0.1](https://pypi.org/project/apache-flink/2.0.1/)
+
+# Release Notes
+
+        Release Notes - Flink - Version 2.0.1
+
+<h2>        Sub-task
```

Comment:
It is better to classify these jira in `Subtask and Technical Debt` category to bug or improvement.